### ISSUE TITLE
Add keyboard navigation to this sim

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/faradays-law#80]
We would like to add keyboard navigation to this sim. @jbphet since you are the responsible dev, does it seem OK to proceed with a11y in this sim or are you aware of anything in this sim that would make keyboard nav challenging?

@zepumph would you please work on implementation of this?  Most of the view in this sim uses common components which have already been instrumented for keyboard navigation. The magnet should be the only thing that requires custom work, and I am hoping we can do something very similar to what was done in friction.

**Comment by jessegreenberg** [comment]
Here is the requested keyboard navigation order:

-  Magnet
- Field lines check  box
- flip magnet  button
- reset all button

**Comment by jessegreenberg** [comment]
[Created a design doc](https://docs.google.com/document/d/1XAGmfk69mEd9Nta8c3kqjCSP0_qZ5DH6uGOMkfDxRuk/edit#), thanks for the template @terracoda!

**Comment by zepumph** [comment]
@jessegreenberg what is the timeline on this? Phet-iO is expecting to do a fair amount of work on this repo in the near future as part of https://github.com/phetsims/faradays-law/issues/79. There is a good chance that I will be working on a lot of that issue as well, but I want to make sure that a11y and phet-io don't step on each others' toes.

**Comment by jessegreenberg** [comment]
@zepumph since  this is a single screen sim and most of the UI is already interactive with the keyboard, we were hoping that a minor release of this sim could be deployed with keyboard nav (without description) relatively quickly. But I don't know exact timelines and if it is going to have some phet-io work soon that should also be considered. Can we discuss at tomorrow's meeting?

**Comment by zepumph** [comment]
Sounds good. I'll poke around in it tonight before the meeting.

**Comment by jessegreenberg** [comment]
Thanks @zepumph!

**Comment by zepumph** [comment]
With @jessegreenberg's help I seemed to get keyboard nav into FL with a single commit. @emily-phet @terracoda I would love some feedback on this version http://www.colorado.edu/physics/phet/dev/html/faradays-law/1.4.0-dev.3/faradays-law_en.html?accessibility

I think that we will need to two things, but I defer to the fine designing judgment of the team.
1. Tweak the magnet positioning function so that it doesn't jump when colliding with coils.
1. Update the focus highlight when the arrows around the magnet disappear.

**Comment by zepumph** [comment]
@jbphet I do apologize. I just realized that I went ahead without waiting for your response. I did minimal work in the repo in the above commit. Please let me know if there are concerns.

**Comment by jbphet** [comment]
I don't have any concerns about adding a11y to this sim.

**Comment by zepumph** [comment]
@terracoda brought up a good point in the keyboard nav meeting today. 

* The focus highlight should go in the coils just like the magnet.

**Comment by zepumph** [comment]
@terracoda I have done all of the suggestions for this sim. Would you have a look and let me know what else needs to be done? 
http://phettest.colorado.edu/faradays-law/faradays-law_en.html?ea&brand=phet&accessibility

**Comment by terracoda** [comment]
@zepumph, looks good!

By the way, I noticed that there is some funny bounciness of the magent even when moving it around with the mouse. The funny bouncing that happens when using the keyboard is not an issue on its own. The visual affects manifest differently (when using keyboard the magnet can disappear into the coil), but there is weird bounciness in both keyboard and mouse use.

**Comment by terracoda** [comment]
@zepumph reassign me if you need me to have another look.

**Comment by zepumph** [comment]
@terracoda your above comment will be covered in #81. Closing

**Comment by zepumph** [comment]
Recent version from today:
http://www.colorado.edu/physics/phet/dev/html/faradays-law/1.4.0-dev.4/faradays-law_en.html?accessibility
